google/cloud/storage/benchmarks/benchmark_utils.cc

```diff
@@ -127,10 +127,9 @@ std::chrono::seconds ParseDuration(std::string const& val) {
   return std::chrono::seconds(s);
 }
 
-google::cloud::StatusOr<bool> ParseBoolean(std::string const& val,
-                                           bool default_value) {
+google::cloud::optional<bool> ParseBoolean(std::string const& val) {
   if (val.empty()) {
-    return default_value;
+    return google::cloud::optional<bool>{};
```

Comment:
Does `return {};` work? If so, it may be nice to use that here and at the end of the function.